server/controllers/configurationsController/index.js

```diff
@@ -0,0 +1,59 @@
+import ConfigModel from '../../models/ConfigModel'
+
+const ConfigurationsController = {
+  create: async (req, res) => {
+    try {
+      const { appDb } = req.app.locals
+      const user = await ConfigModel.save(appDb, req.body)
+      console.log(user, 'WHAT IS THIS')
+
+      return res.status(200).json({ data: user })
+    } catch (error) {
+      return res.status(400).json(error)
+    }
+  },
+  destroy: async (req, res) => {
+    try {
+      const { appDb } = req.app.locals
+      const { config_id } = req.params
+      const { deletedCount } = await ConfigModel.destroy(appDb, config_id)
+
+      if (deletedCount > 0) {
+        return res.status(200)
+      } else {
+        return res.status(404)
+      }
+    } catch (error) {
+      res.status(400).json(error)
+    }
+  },
+  edit: async (req, res) => {
+    try {
+      const { appDb } = req.app.locals
+      const { config_id } = req.params
+      const { value } = await ConfigModel.edit(appDb, config_id, req.body)
+
+      if (value) {
+        return res.status(200).json({ data: value })
+      } else
+        return res
+          .status(404)
+          .json({ error: new Error('Unable to edit configuration') })
+    } catch (error) {
+      return res.status(400).json({ error })
+    }
+  },
+  index: async (req, res) => {
+    try {
+      const { appDb } = req.app.locals
+      const { uid } = req.params
+      const data = await ConfigModel.index(appDb, uid)
+
+      return res.status(200).json(data)
```

Comment:
the other actions render the data in an object with a `data` key. Let's be consistent with the response objects.

Reply:
[f11e811](https://github.com/AMP-SCZ/dpdash/pull/305/commits/f11e811a37bb7df8cc158a8d0e061615e8ec535d)